server/src/test/java/org/opentosca/toscana/plugins/scripts/BashScriptTest.java

```diff
@@ -0,0 +1,66 @@
+package org.opentosca.toscana.plugins.scripts;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.opentosca.toscana.core.BaseJUnitTest;
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+import org.opentosca.toscana.core.transformation.logging.Log;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class BashScriptTest extends BaseJUnitTest {
+
+    private BashScript bashScript;
+    private PluginFileAccess access;
+    private File targetScriptFolder;
+
+    @Before
+    public void setUp() {
+        bashScript = null;
+        access = new PluginFileAccess(tmpdir, tmpdir, mock(Log.class));
+        try {
+            bashScript = new BashScript(access, "test");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        targetScriptFolder = new File(tmpdir, "content/scripts/");
```

Comment:
another hard coded string. Please find the correlated String in production code and introduce a constant.